src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -0,0 +1,406 @@
+<template>
```

Comment:
We should probably convert this into a `kv-carousel`. The original homepage loan carousel was built before `kv-carousel`, and hasn't been swapped over yet.

Reply:
I did wonder about that and it's an improvement we can make if there is time, but it was already working an I wanted to start this prototyping work...